client/components/banner/test/index.jsx

```diff
@@ -1,165 +1,197 @@
 	} );
 
 	test( 'should render a <PlanPrice /> when price is specified', () => {
-		const comp = shallow( <Banner { ...props } price={ 100 } /> );
-		expect( comp.find( PlanPrice ) ).toHaveLength( 1 );
+		render( <Banner { ...props } price={ 100 } /> );
+		expect( screen.getByText( '100' ) ).toBeVisible();
 	} );
 
 	test( 'should render two <PlanPrice /> components when there are two prices', () => {
-		const comp = shallow( <Banner { ...props } price={ [ 100, 80 ] } /> );
-		expect( comp.find( PlanPrice ) ).toHaveLength( 2 );
+		render( <Banner { ...props } price={ [ 100, 80 ] } /> );
+		expect( screen.getByText( '100' ) ).toBeVisible();
+		expect( screen.getByText( '80' ) ).toBeVisible();
 	} );
 
 	test( 'should render no <PlanPrice /> components when there are no prices', () => {
-		const comp = shallow( <Banner { ...props } /> );
-		expect( comp.find( PlanPrice ) ).toHaveLength( 0 );
+		const { container } = render( <Banner { ...props } /> );
+		expect( container.getElementsByClassName( 'plan-price__integer' ) ).toHaveLength( 0 );
 	} );
 
 	test( 'should render a .banner__description when description is specified', () => {
-		const comp = shallow( <Banner { ...props } description="test" /> );
-		expect( comp.find( '.banner__description' ) ).toHaveLength( 1 );
+		render( <Banner { ...props } description="test" /> );
+		expect( screen.getByText( 'test' ) ).toBeVisible();
 	} );
 
 	test( 'should not render a .banner__description when description is not specified', () => {
-		const comp = shallow( <Banner { ...props } /> );
-		expect( comp.find( '.banner__description' ) ).toHaveLength( 0 );
+		const { container } = render( <Banner { ...props } /> );
+		expect( container.getElementsByClassName( 'banner__description"' ) ).toHaveLength( 0 );
 	} );
 
 	test( 'should render a .banner__list when list is specified', () => {
-		const comp = shallow( <Banner { ...props } list={ [ 'test1', 'test2' ] } /> );
-		expect( comp.find( '.banner__list' ) ).toHaveLength( 1 );
-		expect( comp.find( '.banner__list li' ) ).toHaveLength( 2 );
-		expect( comp.find( '.banner__list li' ).at( 0 ).text() ).toContain( 'test1' );
-		expect( comp.find( '.banner__list li' ).at( 1 ).text() ).toContain( 'test2' );
+		const { container } = render( <Banner { ...props } list={ [ 'test1', 'test2' ] } /> );
+		expect( container.querySelectorAll( '.banner__list' ) ).toHaveLength( 1 );
+		expect( container.querySelectorAll( '.banner__list li' ) ).toHaveLength( 2 );
+		expect( screen.getByText( 'test1' ) ).toBeVisible();
+		expect( screen.getByText( 'test2' ) ).toBeVisible();
 	} );
 
 	test( 'should not render a .banner__list when description is not specified', () => {
-		const comp = shallow( <Banner { ...props } /> );
-		expect( comp.find( '.banner__list' ) ).toHaveLength( 0 );
+		const { container } = render( <Banner { ...props } /> );
+		expect( container.getElementsByClassName( '.banner__list' ) ).toHaveLength( 0 );
```

Comment:
We should remove the dot here.

```suggestion
		expect( container.getElementsByClassName( 'banner__list' ) ).toHaveLength( 0 );
```

Although we could change this to query by the `list` role, and change the matcher to be `.not.toBeInTheDocument()`.